src/common/src/util/scan_range.rs

```diff
@@ -72,6 +72,69 @@ impl ScanRange {
         }
     }
 
+    /// `Precondition`: make sure the first order key is int if you call this method.
```

Comment:
Can we add a assert?

Reply:
ScanRange itself do not contain any table column type information.